### ISSUE TITLE
Add Windows 11 and Windows Server 2022

### ISSUE THREAD

**Issue by mmoioli** [MicrosoftDocs/msix-docs#278]
MSIX is supported for sure on Windows 11, not 100% sure about Windows Server 2022 but our colleague Thomas Maurer was able to install a MSIX package also on 2022.
https://www.thomasmaurer.ch/2021/05/install-windows-terminal-on-windows-server-2022/

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 95b83f04-01b6-c494-1e2a-dcddecaa7dc8
* Version Independent ID: 758db7a1-fd48-56fc-1e82-0fbd7ad3f7dc
* Content: [Supported platforms - MSIX](https://docs.microsoft.com/en-us/windows/msix/supported-platforms)
* Content Source: [msix-src/supported-platforms.md](https://github.com/microsoftdocs/msix-docs/blob/master/msix-src/supported-platforms.md)
* Product: **msix**
* GitHub Login: @dianmsft
* Microsoft Alias: **diahar**

**Comment by anarvekar-msft** [comment]
@mmoioli Yes, MSIX is supported on 2022. The docs will be updated for this soon.
